fairlearn/reductions/_exponentiated_gradient/exponentiated_gradient.py

```diff
@@ -51,17 +51,21 @@ class ExponentiatedGradient(BaseEstimator, MetaEstimatorMixin):
         if True each step of exponentiated gradient is followed by the saddle
         point optimization over the convex hull of classifiers returned so
         far; default True
+    sample_weight_key : str
+        Argument to supply `sample_weight` in `estimator.fit`.
```

Comment:
See above for suggested change

Reply:
Similarly, I think we should change `sample_weight_key` to `sample_weight_name`.